platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/ResolutionIssuesIntegrationTest.groovy

```diff
@@ -577,235 +396,6 @@ class ResolutionIssuesIntegrationTest extends AbstractIntegrationSpec {
         outputContains("[producer2-bar, producer1-foo]")
     }
 
-    def "graphs causes unattached constraint edge to exist when module goes back to pending"() {
```

Comment:
Moved to `ResolutionIssuesRegressionIntegrationTest`. This file should only contain _unresolved_ issues